airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java

```diff
@@ -255,6 +256,7 @@ void testReplicationFailure() {
   }
 
   @Test
+  @Disabled
```

Comment:
do we need to disable all these? I think we can just remove verifyShouldRefreshSchema and verifyRefreshSchema

Reply:
@alovew You are correct.  We figured that out after doing the blanket disable.  I believe that we only ended up disabling one test and just commented out the references to those verifications in the others.